test.sh

```diff
@@ -1,4 +1,3 @@
 #!/bin/bash
 
-docker-compose -f test/docker-compose.py2.yml up --build
-docker-compose -f test/docker-compose.py3.yml up --build
+docker-compose -f test/docker-compose.test.yml up --build
```

Comment:
would be nice to add a `-p` flag here.
This command is a bit simplistic, it doesn't exit when the tests finish running. Can we use `docker-compose run` instead?